IS0402Test.py

```diff
@@ -556,7 +556,8 @@ def test_21_2(self, test):
         description = "test_21_2"
 
         # Initial test cases based on the examples in NMOS documentation
-        # See https://github.com/AMWA-TV/nmos-discovery-registration/blob/v1.2.x/docs/2.5.%20APIs%20-%20Query%20Parameters.md#pagination
+        # See https://github.com/AMWA-TV/nmos-discovery-registration/blob/v1.2.x/docs/2.5.%20APIs%20-%20Query%20
+        # Parameters.md#pagination
```

Comment:
Please, no.

Reply:
I'm not sure what to do as an alternative without breaching the line length...

Reply:
Looks like we either add `# noqa: E501` (or whatever code it is) at the end of the line (see [in-line-ignoring-errors](http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors)), or put the word `See` on its own line or omit it entirely (see [max-line-length](http://flake8.pycqa.org/en/3.1.1/user/options.html?highlight=line%20length#cmdoption-flake8--max-line-length)).